test/libsolidity/syntaxTests/constructor/constructor_no_visibility.sol

```diff
@@ -1,3 +1,2 @@
 contract A { constructor() {} }
 // ----
```

Comment:
This test case can be deleted. It is a repetition of test/libsolidity/syntaxTests/constructor/constructor.sol and doesn't add anything new.